assembly/std/index.ts

```diff
@@ -140,28 +236,42 @@ export function generateEvent(event: string): void {
 /**
  * Transfers SCE coins from the current address to given address.
```

Comment:
I would not use abreviation in function description. it can make harder for developers to understand the purpose of the function.